### ISSUE TITLE
Project dead?

### ISSUE THREAD

**Issue by sebthom** [dockupdater/dockupdater#42]
@DirtyCajunRice @harcher81 @mathcantin

Hey guys,

I am wondering what the official project status is, since activity on bug reports and PRs happens for more than half a year.

Can we expect any updates in the near future?

If not then I am thinking about maintaining a fork. I can also help out here.

Thanks!

**Comment by sebthom** [comment]
@DirtyCajunRice thanks for your response, I just saw you as one of the main contributors in the statistics that's why I tagged you. 

interesting to know that this is a hard fork.

sorry for bothering you!

still hoping to hear from the maintainer(s).

**Comment by kcrkor** [comment]
Still no news from the maintainers it seems.

Does reviewing & approving the pull requests allow for automatic merging?

It seems there are two approvals required like in this request: #30

**Comment by sebthom** [comment]
@kcrkor Yeah this project is dead. The project owner didn't show any interest in getting other contributors on board. It's a pity though that he didn't even see the need to properly sunset the project and just left it dangling around.

Anyways as Nichlas pointed out, this project was a hard fork of his now inactive https://github.com/pyouroboros/ouroboros project. However another soft fork exists which seems to be quite active: https://github.com/gmt2001/ouroboros

**Comment by Enissay** [comment]
Since this is a hard fork from [ouroboros](https://github.com/gmt2001/ouroboros), could someone fork it to keep it maintained. I see many pending Pull requests which could improve the project. Many more would come I guess once it is revived.
I would've done it myself if I was not a `python` ignorant :x
